deploy/transport-controller-deployment.yaml.template

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: transport-controller-config
+  namespace: ${WDS_NAME}-system
+data:
+  get-kubeconfig.sh: |
+    #!/bin/bash
+    # this script receives a ControlPlane name and a parameter
+    # that determines whether to extract the ControlPlane's in-cluster kubeconfig
+    # or the external kubeconfig (if set to "true", the first will be retrieved).
+    # The function returns the requested kubeconfig's content in base64.
+    # it assumes that it runs within a kubeflex context.
+
+    cpname="$1"
+    get_incluster_key="$2"
+
+    key=""
+    if [[ $get_incluster_key == "true" ]]; then
```

Comment:
I would quote the LHS here, to protect against the case where it has spaces in it.

Reply:
the LHS is just `true` or `false`, which represents: `true` for using in cluster, `false` for external kubeconfig.
I can add it if you insist, but don't see much value.

Reply:
While the second arg to this script _should_ be `true` or `false`, it is generally better if a script is prepared for the caller to goof up.

Reply:
fixed